src/front/styles/footer.css

```diff
@@ -0,0 +1,35 @@
+.page-footer {
+  background-color: #ffb74d !important;
+  color: white;
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+}
+
+a {
+  color: white;
+}
+
+#signature {
+  font-size: 13px;
+}
+
+.fa {
+  width: 50px;
+  height: 50px;
+}
+
+#links {
+  padding-bottom: 45px;
+}
+
+#icons-container {
+  position: relative;
+}
+
+#icons {
+  position: absolute;
+  bottom: 0%;
+  left: 0%;
```

Comment:
I don't know if I got what you tried to do here, but if it did what you wanted, then I'm happy w/ it as well! 😌 🥂